packages/components/e2e/component-objects/src/main/java/org/talend/component/AppHeaderBar.java

```diff
@@ -12,9 +12,9 @@
  */
 public class AppHeaderBar extends Component {
 
-    private static final String NAME = "HeaderBar";
+    private static final String HEADER_BAR_NAME = "HeaderBar";
 
-    private static final String SELECTOR = ".tc-header-bar";
+    private static final String HEADER_BAR_SELECTOR = ".tc-header-bar";
```

Comment:
No need to repeat `HEADER_BAR` since you are already inside the `AppHeaderBar` class.